department-of-veterans-affairs/caseflow
# Kev ma/appeals 46956 v1

Resolves [42462](https://jira.devops.va.gov/browse/APPEALS-42462)

# Description
Refactored VBMS Service GetDocumentContent SOAP request with ClaimEvidence REST API getDocumentContent GET request to /files/{series_id}/content when :use_ce_api FeatureToggle is enabled

## Acceptance Criteria

## Testing Plan
1. Open rails console in sandbox mode so database changes do not persist
2. Note in Caseflow UAT environment the result of running of Document.fetch_and_cache_document _from_vbms
3. Note result of PrepareEstablishClaims job
4. Close the console and re-open the console to rollback temporary changes
5. Enable :use_ce_api FeatureToggle in UAT
6. Ensure that Document.fetch_and_cache_document returns the same file
7. Ensure that the PrepareEstablishClaims job return the same result

# Backend
## Integrations: Adding endpoints for external APIs

# Best practices
## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
